doc/admin-guide/files/records.config.en.rst

```diff
@@ -317,11 +317,24 @@ Thread Variables
    The number of threads |TS| will create if `proxy.config.exec_thread.autoconfig`
    is set to ``0``, otherwise this option is ignored.
 
+.. ts:cv:: CONFIG proxy.config.exec_thread.listen INT 0
+
+   If enabled (``1``) all the exex_threads listen for incoming connections. `proxy.config.accept_threads`
+   should be disabled to enable this variable.
+
 .. ts:cv:: CONFIG proxy.config.accept_threads INT 1
 
    The number of accept threads. If disabled (``0``), then accepts will be done
    in each of the worker threads.
 
+   ==============  ==================   ===================
+   accept_threads  exec_thread.listen   Effect
+   ==============  ==================   ===================
+   ``0``                 ``0``          All worker threads accept new connections and share listen fd.
+   ``1``                 ``0``          New conections are accepted on a dedicated accept thread and distributed to worker threads in round robin fashion.
+   ``0``                 ``1``          All worker threads listen on the same port using SO_REUSEPORT.
+                                        Each thread has its own listen and new connections are accepted on all the threads.
```

Comment:
Which one is the default?

Reply:
default is 1(accept_threads) 0(listen). We should probably mention that.

Reply:
updated.